Internals: check move warning

@gezalore minor thing, when I run 'make clang-tidy` (which is far from clean) I get these two.  I'm not up on when move is legal, can you please see if they are suspect?

```
src/V3DfgVertices.h:187:9: warning: Method called on moved-from object 'm_driverData' of type 'std::vector' [clang-analyzer-cplusplus.Move]
        m_driverData.emplace_back(flp, index);
        ^
src/V3DfgDecomposition.cpp:532:12: note: Calling 'ExtractCyclicComponents::apply'
    return ExtractCyclicComponents::apply(*this, label);
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/V3DfgDecomposition.cpp:527:26: note: Calling constructor for 'ExtractCyclicComponents'
        return std::move(ExtractCyclicComponents{dfg, label}.m_components);
                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/V3DfgDecomposition.cpp:511:34: note: Assuming the condition is false
        , m_prefix{dfg.name() + (label.empty() ? "" : "-") + label + "-component-"} {
                                 ^~~~~~~~~~~~~




./src/V3DfgVertices.h:246:9: warning: Method called on moved-from object 'm_driverData' of type 'std::vector' [clang-analyzer-cplusplus.Move]
        m_driverData.emplace_back(flp, lsb);
        ^
src/V3DfgDecomposition.cpp:532:12: note: Calling 'ExtractCyclicComponents::apply'
    return ExtractCyclicComponents::apply(*this, label);
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/V3DfgDecomposition.cpp:527:26: note: Calling constructor for 'ExtractCyclicComponents'
        return std::move(ExtractCyclicComponents{dfg, label}.m_components);
                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/src/V3DfgVertices.h
+++ b/src/V3DfgVertices.h
@@ -196,7 +196,8 @@ public:
     // Remove undriven sources
     void packSources() {
         // Grab and reset the driver data
-        std::vector<DriverData> driverData{std::move(m_driverData)};
+        std::vector<DriverData> driverData;
+        driverData.swap(m_driverData);
 
         // Grab and unlink the sources
         std::vector<DfgVertex*> sources{arity()};
@@ -255,7 +256,8 @@ public:
     // Remove undriven sources
     void packSources() {
         // Grab and reset the driver data
-        std::vector<DriverData> driverData{std::move(m_driverData)};
+        std::vector<DriverData> driverData;
+        driverData.swap(m_driverData);
 
         // Grab and unlink the sources
         std::vector<DfgVertex*> sources{arity()};

